packages/wallet/frontend/src/pages/account/[accountId].tsx

```diff
@@ -202,10 +202,12 @@ export const getServerSideProps: GetServerSideProps<{
     }
   }
 
-  const paymentPointers = paymentPointersResponse.data.map((pp) => ({
-    ...pp,
-    url: pp.url.replace('https://', '$')
-  }))
+  const paymentPointers = paymentPointersResponse.data.paymentPointers
+    .concat(paymentPointersResponse.data.wmPaymentPointers)
```

Comment:
In the accounts page, why do we also show in the list of PPs the WM PPs? 
I am asking because there will be a separate list for WM PPs. So why do we show them in the general PP list as well?

Reply:
Read my comment above:
https://github.com/interledger/testnet/pull/896#issuecomment-1768372919

Reply:
this is just so the frontend does not crash on the response, I am expecting the client code on what to do with the two lists to be in a separate PR

Reply:
@Tymmmy  do you want me to also remove the wmPaymentPointers here?